benchmark/util/text-decoder.js

```diff
@@ -31,7 +32,9 @@ function main({ encoding, len, n, ignoreBOM, type }) {
 
   bench.start();
   for (let i = 0; i < n; i++) {
-    decoder.decode(buf);
+    try {
```

Comment:
Why the try catch?

Reply:
fatal options throw an error on invalid utf8 sequences. in order to continue the benchmark, try catch is required.